src/Morphir/SpringBoot/Backend.elm

```diff
@@ -568,8 +571,11 @@ mapStatefulAppDefinition currentPackagePath currentModulePath =
                                 [ TypeVar "K", TypeVar "C", TypeVar "S", TypeVar "E" ]
                             , ctorArgs =
                                 [ [ ArgDecl []
-                                        (FunctionType (TupleType [ TypeVar "morphir.sdk.Maybe.Maybe[S]", TypeVar "C" ])
-                                            (TupleType [ TypeVar "morphir.sdk.Maybe.Maybe[S]", TypeVar "E" ])
+                                        (FunctionType (Scala.TypeApply (Scala.TypeRef [ "morphir", "sdk", "Maybe" ] "Maybe") [ Scala.TypeVar "S" ])
+                                            (FunctionType
+                                                (TypeVar "C")
+                                                (TupleType [ TypeVar "morphir.sdk.Maybe.Maybe[S]", TypeVar "E" ])
```

Comment:
Can you please break up the `TypeVar "morphir.sdk.Maybe.Maybe[S]"` piece as well?

Reply:
Done